server/routes/profile.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const multer = require("multer");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { savePhoto } = require("../controllers/profile");
+
+const upload = multer({dest: './uploads'});
+
+// router.route("/photo").post(protect, savePhoto);
```

Comment:
unused code?

Reply:
It will be used for ticket #41.

Reply:
It is a good practice to add a 'TODO comment' with a description along with commented code :)